sympy/stats/tests/test_continuous_rv.py

```diff
@@ -458,6 +490,8 @@ def test_studentt():
 
     X = StudentT('x', nu)
     assert density(X)(x) == (1 + x**2/nu)**(-nu/2 - 1/2)/(sqrt(nu)*beta(1/2, nu/2))
+    assert cdf(X)(x) == 1/2 + x*gamma(nu/2 + 1/2)*hyper((1/2, nu/2 + 1/2),
```

Comment:
A problem with these tests (both for cdf and for density) is that they are essentially the same code pasted twice. They don't really check how the formulas __work__, they only check what the formulas __are__.  Try to add a test that involves  
`diff(cdf(X)(x), x)`  compared to `density(X)(x)`. That would make sure that the two things are consistent: the derivative of the CDF is the PDF. 

It's quite possible that `Eq(diff(cdf(X)(x), x), density(X)(x)).simplify()` will fail to simplify to True, because of how complex the formulas are. But perhaps you can do `.subs(x, Rational(random.gauss(0, 1)))` in there, verifying the equality at a randomly chosen rational point. Worth trying. 
  
There is already `test_precomputed_cdf` but it is based on integrating the PDF and we know that won't work well because these PDF can't be integrated easily. But taking the derivative of cdf should be easier.

Reply:
Should I add this test for every method or just for the ones I have added?

Reply:
I imagine this could be one test running in a loop over distributions. See how test_precomputed_cdf is written.